wrapper/src/test/java/integration/refactored/host/TestEnvironment.java

```diff
@@ -617,12 +649,62 @@ private static String getEnvironmentInfoAsString(TestEnvironment env) {
 
   public void runTests(String taskName) throws IOException, InterruptedException {
     final ContainerHelper containerHelper = new ContainerHelper();
-    containerHelper.runTest(this.testContainer, taskName);
+
+    if (this.info.getRequest().getFeatures().contains(TestEnvironmentFeatures.RUN_HIBERNATE_TESTS_ONLY)) {
+      final Long exitCode = containerHelper.runCmdInDirectory(
+          this.testContainer,
+          "/app/hibernate-orm",
+          buildHibernateCommands(false));
+      containerHelper.runCmd(this.testContainer, "./collect_test_results.sh");
+      assertEquals(0, exitCode, "failed to run Hibernate ORM tests");
```

Comment:
Would "Hibernate ORM tests failed" be more accurate? Or does an exit code of 0 always mean the tests weren't even run?